specs/change_password/02_user_interface.rst

```diff
@@ -0,0 +1,136 @@
+=====================
+User interface
+=====================
+
+=====  ===============  =========  ==========
+epic        title        author     created
+=====  ===============  =========  ==========
+`74`_  Change password  tschubotz  2019-01-17
+=====  ===============  =========  ==========
+
+.. _74: https://github.com/gnosis/safe/issues/74
+
+.. _Main:
+
+
+#. `Main`_
+#. `User Interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Current password
+------------------
+
+User has to enter the current password.
+
+.. raw:: html
+
+   <img src="screens/android/current_password.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/current_password.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/bJK86GW
+* Zeplin link iOS: https://zpl.io/aNP44ZQ
+
+Current password - Invalid
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+User has not entered the current password correctly.
+
+.. raw:: html
+
+   <img src="screens/android/current_password_invalid.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/current_password_invalid.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/aBEv3KK
+* Zeplin link iOS: https://zpl.io/b61lQOd
+
+
+New password
+------------------
+
+User has to enter new password and confirm.
+
+.. raw:: html
+
+   <img src="screens/android/new_password.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/new_password.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/aR1NNXK
+* Zeplin link iOS: https://zpl.io/bJ7NOeW
+
+New password - Invalid
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+User has entered a new password that does not meet the requirements.
+
+.. raw:: html
+
+   <img src="screens/android/new_password_invalid.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/new_password_invalid.png" width="320px"/>
+
+* Zeplin link Android: https://zpl.io/aR1NNXK
+* Zeplin link iOS: https://zpl.io/bJ7NOeW
```

Comment:
Both Zeplin links point to the screens where there are no errors (does not match the ones from the screenshots)

Reply:
lol Michel essentially changed the screen completely and now the links are fucked. Man this process is realy not nice.